numpy/__init__.pyi

```diff
@@ -1049,7 +1049,7 @@ class _SupportsFileMethods(SupportsFlush, Protocol):
 
 @type_check_only
 class _SupportsFileMethodsRW(SupportsWrite[bytes], _SupportsFileMethods, Protocol):
-    pass
+    ...
```

Comment:
I seem to recall this was automatically fixed by ruff. Or maybe not 🤔? Let me see.

On the _main_ branch:

```console
$ ruff check --select PYI009 numpy/__init__.pyi --fix
Found 2 errors (2 fixed, 0 remaining).
$ 
$ git diff
diff --git a/numpy/__init__.pyi b/numpy/__init__.pyi
index a81e060f62..4a2b110712 100644
--- a/numpy/__init__.pyi
+++ b/numpy/__init__.pyi
@@ -1052,7 +1052,7 @@ def seek(self, offset: int, whence: int, /) -> object: ...
 
 @type_check_only
 class _SupportsFileMethodsRW(SupportsWrite[bytes], _SupportsFileMethods, Protocol):
-    pass
+    ...
 
 @type_check_only
 class _SupportsItem(Protocol[_T_co]):
@@ -1069,7 +1069,7 @@ def shape(self, /) -> _ShapeT_co: ...
 
 @type_check_only
 class _HasShapeAndSupportsItem(_HasShape[_ShapeT_co], _SupportsItem[_T_co], Protocol[_ShapeT_co, _T_co]):
-    pass
+    ...
 
 # matches any `x` on `x.type.item() -> _T_co`, e.g. `dtype[np.int8]` gives `_T_co: int`
 @type_check_only
$ 
```

So that's ruff, not me.

Reply:
strange, the python typing spec style guide want it to be inline:
https://typing.readthedocs.io/en/latest/guides/writing_stubs.html#blank-lines

Reply:
If you put the ellipses at the end of the line, ruff will leave them there. I expect what is happening here is simply that they are replacing `pass`, which is on the next line.

Reply:
Yes, exactly.